app/views/rubygems/_rubygem.html.erb

```diff
@@ -1,10 +1,10 @@
-<%= link_to rubygem_path(rubygem), :class => 'gems__gem' do %>
+<%= link_to "/gems/#{rubygem.name}", :class => 'gems__gem' do %>
```

Comment:
route helper would only work on activerecord object and not the response returned by ES. We validate `name` against `[A-Za-z0-9.-_]`, so this shouldn't lead to any unintended behavior.